packages/components/src/Form/Inputs/Checkbox/Checkbox.tsx

```diff
@@ -122,6 +125,20 @@ export const CheckboxContainer = styled.div<CheckboxContainerProps>`
           `
     }}
     color: ${({ theme }) => theme.colors.palette.white};
+    ${(props) =>
+      props.validationType === 'error'
+        ? `
+      border-color: ${props.theme.colors.palette.red400};
+      &:hover {
+        border-color: ${props.theme.colors.palette.red500};
+      }
+      &:focus,
+      :focus-within {
+        border-color: ${props.theme.colors.palette.red500};
+        box-shadow: 0 0 0 2px ${props.theme.colors.palette.red100};
+      }
+      `
+        : ''}
```

Comment:
This should get pulled out as a helper:

```suggestion
      ${({validationType}) => showValidation(validationType)}
```

```js
const showValidation = (validationType: undefined | 'error') => validationType ? css`
  border-color: ${props.theme.colors.palette.red400};

  &:hover {
    border-color: ${props.theme.colors.palette.red500};
  }
    
  &:focus,
  :focus-within {
    border-color: ${props.theme.colors.palette.red500};
    box-shadow: 0 0 0 2px ${props.theme.colors.palette.red100};
  }
` : ''
```

Reply:
actually I can just import. I forgot to fix it after I was playing around with this.
import { inputTextValidation } from '../InputText'